screens/courses/CoursesScreen.js

```diff
@@ -50,20 +49,18 @@ class CoursesScreen extends React.Component {
       date: date,
     });
   }
-  _handleViewStudents(course_id) {
-    this.props.navigation.navigate('Students', {
-      courseId: course_id,
-    })
-  }
 
   _renderCourses() {
-    return this.props.courses.map((course, i) => (
+    <Button
+      onPress={() => navigate('EditCourse', {refreshCourses: this._fetchCourses, newCourse: true, sessions: []})}
```

Comment:
I usually avoid using arrow functions in the render methods because the function gets reconstructed upon every render call, instead of just once at the initialization of the component. It might not make a huge difference, and sometimes you may need to fill in partial arguments so it's probably fine to take the convenience of the arrow func in that case